src/CLI11.hpp

```diff
@@ -0,0 +1,10966 @@
+// CLI11: Version 2.4.1
```

Comment:
Maybe we can add this as a Git submodule?

Reply:
added